cmdstanpy/model.py

```diff
@@ -72,6 +74,12 @@ def __init__(
         self._logger = logger or get_logger()
 
         if model_name is not None:
+            if len(model_name.strip()) == 0:
```

Comment:
```suggestion
            if not model_name.strip():
```

Reply:
Also, do we care if model name contains spaces?

    if not model_name.strip() or (len(model_name.strip().split()) > 1):